cmu_graphics/utils.py

```diff
@@ -113,6 +115,8 @@ def isGroup(shape):
 
 def getChildShapes(shape):
     result = []
+    if hasattr(shape, '_shape'):
```

Comment:
@austin-schick When you passed a group as an argument to Shape.hitsShape(), the child shapes of that group were not converted to shape_logic.Shape. Is this the right way to fix that?

Reply:
This looks like a good solution. How is this working in `cmu-graphics.js` right now? I just took a look and didn't see a conversion happening

Reply:
It's handled by `jsApply`. Maybe the difference is that JS Group objects only hold references to JS Shapes, while in desktop CMU Graphics SLI.Group objects can refer to cmu_graphics.Shapes?